.github/workflows/build-and-test-msi.yaml

```diff
@@ -8,6 +8,8 @@ on:
       ref_name:
         required: true
         type: string
+  schedule:
+    - cron: '0 9 * * *'
```

Comment:
Maybe a comment about running this job at 9:00 UTC will help decoding the cron expression.
# builds windows installer and runs tests daily at 9:00 UTC.